From 07c12383e0ef5645dde3cb0202cfe45fc181ca04 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Thu, 21 May 2015 17:26:42 +0800
Subject: [PATCH] MLK-10951 ARM: dts: imx6ul: correct sdma compatible string

Correct the sdma compatible string since the IP is the same as i.MX6SX,
otherwise the driver match to imx35 IP type, but use imx6q sdma firmware,
which cause sdma cannot work after suspend/resume.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 5fbc330..5a57476 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -695,7 +695,7 @@
 			};
 
 			sdma: sdma@020ec000 {
-				compatible = "fsl,imx6ul-sdma", "fsl,imx35-sdma";
+				compatible = "fsl,imx6sx-sdma", "fsl,imx35-sdma";
 				reg = <0x020ec000 0x4000>;
 				interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6UL_CLK_SDMA>,
-- 
1.8.0

